diff -up kcron-20.03.90/src/crontabWidget.cpp.omv~ kcron-20.03.90/src/crontabWidget.cpp
--- kcron-20.03.90/src/crontabWidget.cpp.omv~	2020-04-13 13:58:05.824788147 +0200
+++ kcron-20.03.90/src/crontabWidget.cpp	2020-04-13 13:58:09.794835628 +0200
@@ -192,11 +192,7 @@ QHBoxLayout* CrontabWidget::createCronSe
 	}
 
 	connect(group, static_cast<void (QButtonGroup::*)(QAbstractButton *)>(&QButtonGroup::buttonClicked), this, &CrontabWidget::refreshCron);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
 	connect(d->otherUsers, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), this, &CrontabWidget::checkOtherUsers);
-#else
-	connect(d->otherUsers, static_cast<void (QComboBox::*)(int, const QString &)>(&QComboBox::currentIndexChanged) , this, &CrontabWidget::checkOtherUsers);
-#endif
 
 	layout->addStretch(1);
 
